src/components/Timestamp.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+
+function Timestamp(props) {
+  const timestamp = props.timestamp;
```

Comment:
Also, here it's better practice when we use object destructuring : const {timestamp} = props;